wsgi.py

```diff
@@ -359,7 +359,15 @@ def get_quota(project):
 def put_quota(project):
     shift = get_openshift()
     moc_quota = request.get_json(force=True)
-    return shift.replace_moc_quota(project, moc_quota)
+    return shift.update_moc_quota(project, moc_quota)
+
+
+@APP.route("/projects/<project>/quota", methods=["PATCH"])
+@AUTH.login_required
+def patch_quota(project):
+    shift = get_openshift()
+    moc_quota = request.json
```

Comment:
Why are we referencing `request.json` here, but `request.get_json(force=True)` in the previous method?

Reply:
possibly because I am not always setting the correct mime type for the data that is being sent, and haven't yet tested the patch function (which will probably also require the get_json(force=True))